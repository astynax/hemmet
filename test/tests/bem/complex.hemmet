form:search-form$theme>input.query>(div.help~hidden_t)+span.submit-button~disabled_t:button~text_small>.hint
