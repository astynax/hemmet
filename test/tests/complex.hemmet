form:search-form$theme>input.query^red-text>(div.help~hidden_t)+span.submit&button~text_small>.hint
