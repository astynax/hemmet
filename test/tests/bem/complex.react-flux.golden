formc_ ("search-form" <> theme) $ do
  inputc_ "search-form__query" $ do
    divc_ "search-form__help search-form__help_hidden_t" $ pure ()
  spanc_ "button button_text_small search-form__submit-button search-form__submit-button_disabled_t" $ do
    divc_ "button__hint" $ pure ()
